TODO.md

```diff
@@ -2,20 +2,15 @@
 
 ## Priority list
 
```

Comment:
Can we add the splitting the migration to it's own stage in the pipeline as a priority task please